ui/src/dashboards/components/OverlayControls.tsx

```diff
@@ -1,11 +1,29 @@
-import React from 'react'
-import PropTypes from 'prop-types'
+import React, {SFC, MouseEvent} from 'react'
 import classnames from 'classnames'
 
-import ConfirmOrCancel from 'shared/components/ConfirmOrCancel'
+import ConfirmOrCancel from 'src/shared/components/ConfirmOrCancel'
 import SourceSelector from 'src/dashboards/components/SourceSelector'
+import {QueryConfig, Source} from 'src/types'
 
-const OverlayControls = ({
+interface SourceOption extends Source {
```

Comment:
what do you think about importing this type from the `SourceSelector` component? or somehow otherwise DRYing it out?

Reply:
I can do that